src/main/scala/ai/starlake/job/sink/bigquery/BigQueryNativeJob.scala

```diff
@@ -145,13 +109,27 @@ class BigQueryNativeJob(override val cliConfig: BigQueryLoadConfig, sql: String)
     sourceURIs: String
   ): LoadJobConfiguration = {
     val loadConfig =
-      LoadJobConfiguration
-        .newBuilder(tableId, sourceURIs.split(",").toList.asJava, formatOptions)
-        .setIgnoreUnknownValues(true)
-        .setCreateDisposition(JobInfo.CreateDisposition.valueOf(cliConfig.createDisposition))
-        .setWriteDisposition(JobInfo.WriteDisposition.valueOf(cliConfig.writeDisposition))
-        .setSchema(bqSchema)
+      LoadJobConfiguration.newBuilder(tableId, sourceURIs.split(",").toList.asJava, formatOptions)
+    configureBqLoad(loadConfig, bqSchema).build()
+  }
+
+  private def bqLoadLocaFileConfig(
+    bqSchema: BQSchema,
+    formatOptions: FormatOptions
+  ): WriteChannelConfiguration = {
+    val loadConfig = WriteChannelConfiguration.newBuilder(tableId).setFormatOptions(formatOptions)
```

Comment:
Do I understand that this way we do not need the tempsGcsBucket anymore ?
So I have to update the docs